packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/pages/search/[[...alias]].jsx

```diff
@@ -18,11 +26,35 @@ export default function SearchPage({ hrefLang, footerMenu, errorMessage }) {
 			/>{' '}
 			<PageHeader title="Search Results" />
 			<div className="mt-12 mx-auto max-w-[50vw]">
-				{errorMessage ? (
-					<p className="text-xl text-center">
-						⚠️Unable to fetch your search results⚠️
-					</p>
-				) : null}
+				{errorMessage || !searchResults ? (
```

Comment:
I wouldn't couple the missing search results with the error. I would expect either one of the other to be true at a given time. I would expect a message to render when there are no errors but also no search results because right now there is just a blank screen.
It's probably easier to check for `searchResults.length > 0`. and render "No results found" in that case.